### DISCUSSION TITLE
Code

### DISCUSSION THREAD

**Discussion by atikjaz** [spencerwooo/onedrive-vercel-index#661] [Q&A] [answered]
How can we add a javascript code in body? I want to customise.

**Comment by myl7** [comment, marked as answer]
As a Next.js project, you can follow React / Next.js guides to add your logic.
For example, if you just want to add something like `<script src="...">` for all pages, you can add it into [`pages/_document.tsx`](https://github.com/spencerwooo/onedrive-vercel-index/blob/main/pages/_document.tsx).
